src/redux/books/books.js

```diff
@@ -0,0 +1,31 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const initialState = [];
+const booksReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ADD_BOOK':
+      return [
+        ...state,
+        {
+          id: action.id,
+          titile: action.payload.title,
+          author: action.payload.author,
```

Comment:
-  Kindly make sure to import createSlice from the redux toolkit, to create reducers as per the project requirements :recycle: